<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Button 1</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tool:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eraser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorPreview">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setColor">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>71</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shortcut:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="penCombo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Slot 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="keyLine">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="penBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="keyBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mouseBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="mouseCombo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Middle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Button 2</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tool:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toolBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eraser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorPreview_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setColor_2">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>71</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shortcut:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="penCombo_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Slot 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="keyLine_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="penBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="keyBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mouseBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="mouseCombo_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Middle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Button 3</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tool:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toolBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eraser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorPreview_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setColor_3">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>71</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shortcut:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="penCombo_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Slot 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slot 3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="keyLine_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="penBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="keyBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mouseBox_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="mouseCombo_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Left</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Middle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
